src/main/java/com/example/event_manager/repo/EventRepo.java

```diff
@@ -1,10 +1,17 @@
 package com.example.event_manager.repo;
 
 import com.example.event_manager.model.Event;
+import java.time.LocalDateTime;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface EventRepo extends JpaRepository<Event, Long> {
 
+  List<Event> findAllByDateTimeIsBefore(LocalDateTime localDateTime);
```

Comment:
1. As for now, no. 2. What's the point behind having final methods args in interface ?

Reply:
1. Ok
2. If u use IDE to implement interface it will get finals automatically if u dont use any plugin like save action. As this is spring-data-interface which will not be usual implemented it make no sense, correct.